integration/inspektor-gadget/integration_test.go

```diff
@@ -547,6 +550,7 @@ func TestDns(t *testing.T) {
 				},
 				{
 					Event:      BuildBaseEvent(ns),
+					Comm:       "nslookup",
 					ID:         "0000",
```

Comment:
Not because your changes: Shouldn't we leave this Field Initializer completly empty and set it in the `normalize` function to `""`?

Reply:
Ok. That's how it was done in `integration/local-gadget/trace_dns_test.go` so we can do the same here.